components/notification/notification.service.ts

```diff
@@ -25,39 +27,39 @@ export class NzNotificationService extends NzMNService<NzNotificationContainerCo
 
   success(
     title: string | TemplateRef<void>,
-    content: string | TemplateRef<void>,
+    content: string | TemplateRef<void | { $implicit: NzNotificationComponent; data: NzSafeAny }>,
```

Comment:
I think void is unnecessary here:

```suggestion
    content: string | TemplateRef<{ $implicit: NzNotificationComponent; data: NzSafeAny }>,
```

Reply:
Yes, understanding your point :), i did that to ensure a compatibility with the previous version of Ng Zorro. If we remove this vois type we will have a breaking change.